lib/membrane_rtc_engine/endpoints/hls/audio_mixer_config.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Membrane.RTC.Engine.Endpoint.HLS.AudioMixerConfig do
+  @moduledoc """
+  Module representing audio mixer configuration for the HLS endpoint.
+  """
+
+  @typedoc """
+  * `channels` - Amount of channels inside a frame.
+  * `sample_rate` - Sample rate of the audio.
+  * `sample_format` - Format of samples.
```

Comment:
I am not sure about capital letter here as those words are not at the beginning of bullet points :thinking:  Looking at elixir docs I think we shouldn't use capital letteres after `-` https://hexdocs.pm/elixir/1.14.1/Kernel.html

Could we also provide default values, similarly to CompositorConfig?

```suggestion
  @typedoc """
  * `channels` - number of channels inside a frame.
  * `sample_rate` - sample rate of the audio.
  * `sample_format` - format of samples.
```